packages/charts/src/charts/donut/donut-chart.component.tsx

```diff
@@ -182,6 +186,7 @@ export const DonutChart: FC<Props> = ({
                 }) => (
                   <DonutSlice
                     key={dataKey}
+                    testId={dataKey}
```

Comment:
Rather than using `testId` I suggest to rename it just to `id` and make it required property.